src/modules/create_user/app/create_user_viewmodel.py

```diff
@@ -19,10 +19,13 @@ def __init__(self, user: User):
 
     def to_dict(self):
         return {
-            'user_id': self.user_id,
-            'name': self.name,
-            'email': self.email,
-            'restaurant': str(self.restaurant),
-            'role': self.role.value,
+            'user': {
+                'user_id': self.user_id,
+                'name': self.name,
+                'email': self.email,
+                'restaurant': self.restaurant.value,
```

Comment:
Se for none o restaurante vai dar erro, não?

Reply:
Dudu já commitou arrumando molhas